docs/DeveloperGuide.adoc

```diff
@@ -239,6 +239,128 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+//tag::moduledatabaseadministration[]
+=== Module Database Administration
+
+Module Database Administration are a series of commands that administrators can use to make necessary changes to keep
```

Comment:
Module Database Administration `are` a series of ...
Just to clarify, should this be `is` because there is only one Module Database Administration?